internal/cmd/kafka/command_acl_create.go

```diff
@@ -55,7 +56,19 @@ func (c *aclCommand) create(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	users, userIdMap, err := c.mapResourceIdToUserId(nil)
+	serviceAccounts, err := c.Client.User.GetServiceAccounts(context.Background())
+	if err != nil {
+		return err
+	}
+
+	adminUsers, err := c.Client.User.List(context.Background())
```

Comment:
Are we sure that this function only returns admin users? I thought it returned all users.

Reply:
Pretty sure